src/Array.hpp

```diff
@@ -269,7 +269,7 @@ class Array : public ArrayView< T,
     static_assert( check_dim_type< INDEX_TYPE, DIMS... >::value, "arguments to Array::resize(DIMS...newDims) are incompatible with INDEX_TYPE" );
 
     INDEX_TYPE const oldSize = size();
-    dimUnpack( const_cast< INDEX_TYPE * >( m_dims ), newDims ... );
+    dimUnpack( m_dims, newDims ... );
```

Comment:
when did `m_dims` become an array of non-const? Since the accessors are const, it is OK to not have `ArrayView::m_dims` non-const? Did you run any performance testing for this?

Reply:
When I was doing the array benchmarking, so yeah it didn't effect anything. Pretty much every method in `ArrayView` is const, plus whenever they're captured they're const so it shouldn't make any difference.